docs/content/tutorial/administrative-tasks.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Administrative Tasks"
+date:
+draft: false
+---
+
+## Rotating TLS Certificates
+
+Credentials should be invalidated and replaced (rotated) as often as possible
+to minimize the risk of their misuse. Unlike passwords, every TLS certificate
+has an expiration, so replacing them is inevitable. When you use your own TLS
+certificates with PGO, you are responsible for replacing them appropriately.
+Here's how.
+
+
+PostgreSQL needs to be restarted after its server certificates change.
+There are a few ways to do it:
+
+1. Store the new certificates in a new Secret. Edit the PostgresCluster object
+   to refer to the new Secret and PGO will perform a
+   [rolling restart]({{< relref "/architecture/high-availability.md" >}}#rolling-update).
+   ```yaml
+   spec:
+     customTLSSecret:
+       name: hippo.new.tls
+   ```
+
+2. Replace the old certificates in the current Secret. PGO doesn't notice when
+   the contents of your Secret change, so you need to
+   [trigger a rolling restart]({{< relref "/tutorial/administrative-tasks.md" >}}#manually-restarting-postgresql).
+
```

Comment:
with this being enumerated it feels as though both are to be done vs these are both options 🤔 
same thing for the pgbouncer section.  Otherwise LGTM

Reply:
Agreed. I meant to put an "_or_" between 1. and 2. there. Would that help? 🤔 In the PgBouncer section, that would be (1 or 2 (i or ii)) which is ... a lot of "or". 😬

Reply:
Added some "_or_" and ditched `kubectl rollout` to simplify things. [`39464bf7`](https://github.com/CrunchyData/postgres-operator/pull/2622/commits/39464bf7e06c511e3806ae7f44f40ca6ca4db414)

Reply:
this is less complicated for sure 👍